pkg/schedule/checker/merge_checker_test.go

```diff
@@ -441,6 +443,7 @@ func (suite *mergeCheckerTestSuite) TestMatchPeers() {
 }
 
 func (suite *mergeCheckerTestSuite) TestStoreLimitWithMerge() {
+	re := suite.Require()
```

Comment:
why replace suite with re?

Reply:
Using `Require` will call `FailNow()` to fatal the test as soon as possible rather than calling `Fail()`.